sirepo/server.py

```diff
@@ -615,6 +616,8 @@ def init(uwsgi=None, use_reloader=False):
         _google_tag_manager = f'''<script>
     (function(w,d,s,l,i){{w[l]=w[l]||[];w[l].push({{'gtm.start':new Date().getTime(),event:'gtm.js'}});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);}})(window,document,'script','dataLayer','{cfg.google_tag_manager_id}');
     </script>'''
+# TODO(e-carlin): should this be done here or elsewhere?
+    sirepo.events.init()
```

Comment:
@robnagler Is this the right place for this?

Reply:
It needs to be somewhere "early" so when other modules init and register handlers events is ready.

Reply:
sirepo.events doesn't need to be specially initialized afaict. It can just initialize itself at the module level, just like, say, sirepo.smtp.

Reply:
2c165ad7